express-post-json/server.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+const grades = {};
+
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+  const gradeArr = [];
+  for (const grade in grades) {
+    gradeArr.push(grades[grade]);
+  }
+  res.status(201).json(gradeArr);
+});
+
+app.use(express.json());
```

Comment:
Typically we do this at the top of the file, right after creating `app`. This is because it's easy to get put endpoints that require it before the call, and that leads to hard to find bugs. Also, the JSON middleware does nothing if the content-type is not 'application/json' so there's no harm in having it run on every request.